projects/kit/components/input-range/input-range.style.less

```diff
@@ -1,143 +1,52 @@
-@import 'taiga-ui-local';
-
-@padding: 1rem;
-@padding2: @padding * 2;
-
-:host {
-    .createStackingContext();
-    display: block;
-    font: var(--tui-font-text-m);
-    border-radius: var(--tui-radius-m);
-    color: var(--tui-text-01);
-
-    &._segmented {
-        // TODO: research if it can be used in rem units
-        /* stylelint-disable-next-line */
-        border-bottom: 26px solid transparent;
-    }
-
-    &[data-mode='onDark'] {
-        color: var(--tui-base-01);
+tui-input-number ::ng-deep tui-wrapper {
+    &:after {
```

Comment:
Maybe we can just add this to appearance="none" so we don't resort to ng-deep?

Reply:
✅